### ISSUE TITLE
Can't see preferences icon (Arch Linux)

### ISSUE THREAD

**Issue by gnacho** [voldyman/MarkMyWords#73]
As I'm new user of Apricity (based on Arch - I was using Elementary until now), I realized that preferences icon is not showing well. 
![seleccion_013](https://cloud.githubusercontent.com/assets/10582292/13948011/05d96a2a-f01e-11e5-9756-14fe83b31436.png)

**Comment by emersion** [comment]
That means that the icon is missing from your icon theme.

**Comment by gnacho** [comment]
I tried all icon themes I have installed and none of them are showing this icon :/

**Comment by voldyman** [comment]
what your theme is missing is the `open-menu` icon and the `preferences-system` icon (which is used as fallback). I am not sure if `open-menu` is a standard theme icon so you should talk to the theme creators before reporting a bug. If you want a quick solution get the icons from the [elementary](http://bazaar.launchpad.net/~elementary-design/elementaryicons/trunk/files) theme, minor amount of grepping required

**Comment by gnacho** [comment]
You're right. I was using an old set of Elementary icons. With the new one seems to be perfectly working. Anyway seems to be the only set of icons that show correclty the preferences, maybe should try to use a different (more generic?) icon.

Many thanks for the response again, and for building such an amazing software and release it as open source!! :)

**Comment by voldyman** [comment]
can you confirm from the latest `master` branch?

**Comment by gnacho** [comment]
Tried from here: https://aur.archlinux.org/packages/markmywords-git not sure it compiles the last version but I think it does.

![captura de pantalla de 2016-03-24 11-30-33](https://cloud.githubusercontent.com/assets/10582292/14014624/0ea25170-f1b4-11e5-821b-b00cad52028e.png)

Sorry I'm not a developer, not even engineer, so I can't go further. Thanks for the response again, and for doing this great program.
